.github/workflows/CI.yaml

```diff
@@ -258,6 +258,15 @@ jobs:
           path: ./helm-charts-repository
           token: ${{ secrets.KEPTN_BOT_TOKEN }}
 
+      - name: Substitue local paths with official chart URL
+        if: matrix.config.name == 'keptn'
+        working-directory: ./${{ matrix.config.path }}
+        run: |
+          # substitute local charts path with official ones
+          sed -i 's|file://../.*/chart|https://charts.lifecycle.keptn.sh|' ./Chart.yaml
```

Comment:
I think a more solid option is to use `yq` to change specifically all `repository` fields: https://mikefarah.gitbook.io/yq/operators/assign-update

Reply:
adapted